webui/react/src/shared/styles/antd.scss

```diff
@@ -217,9 +217,12 @@ html {
     opacity: 1;
     user-select: auto;
   }
-  .ant-table .ant-table-tbody > tr > td[class*=ant-table-cell-fix-] {
+  .ant-table .ant-table-tbody > tr > td.ant-table-cell {
     background-color: var(--theme-stage-weak);
```

Comment:
according to theme arch doc, i would think this would be `theme-surface`, as you mentioned in chat.

however i have kept this PR to just target the description overflow.

and we can revisit table color separately if desired.